pipeline/01-train.R

```diff
@@ -233,6 +249,12 @@ if (cv_enable) {
       # nolint end
     )
 
+  # Update the tuning param with the CV range only if early stopping is disabled
+  if (cv_enable && !early_stopping_enable) {
+    lgbm_params <- lgbm_params %>%
+      update(trees = dials::trees(lgbm_range$num_iterations))
+  }
```

Comment:
**[Nitpick, non-blocking]** I think the `cv_enable` check is redundant here since we're already in a conditional block where `cv_enable` is guaranteed to be true starting on line 204, right?

Reply:
Correct! Fixed in https://github.com/ccao-data/model-res-avm/pull/172/commits/d53c9fe2465e1323204d8bdb33da930c029f6e31